resources/assets/uad_lists.json

```
@@ -37724,4 +37724,22 @@
     "labels": [],
     "removal": "Expert"
   }
+}
+  "com.samsung.unifiedtp":{
```

```suggestion
  "com.samsung.unifiedtp": {
```

Without a space it breaks the list.